index.html

```diff
@@ -200,5 +200,37 @@ <h1>About</h1>
                         </div>
                     </div>
                 </section>
+
+                <section class="contactForm">
+                    <div class="contactLogo"><img src="./images/logo.png"
+                            alt="Logo"></div>
+                    <div class="content">
+                        <h1>Get started</h1>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing
+                            elit, sed do eiusmod tempor incididunt ut labore et
+                            dolore magna.</p>
+                    </div>
+                    <form action="https://formspree.io/f/mjvzkrzj"  method="POST">
+                        <input type="text" required id="fullName" placeholder="Your name"
+                            name="Full Name" maxlength="30">
+                        <input type="email" required="required" name="" Email id="myEmail"
+                            placeholder="Your email">
+                        <textarea name="subject" id="msg" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        <input type="submit" value="Start collaboration" id="button">
+                    </form>
+                </section>
+                <section class="myFooter">
+                    <footer>
+                        <h3>Get my Resume <img src="./images/footerImage.png" alt="logo"></h3>
+                        <div class="myHandles">
+                            <a href="https://stackoverflow.com/"><img src="./images/ic_stackoverflow.png" alt="stackflow logo"></a> 
+                            <a href="https://github.com/solorics"><img src="./images/github.png" alt="github logo"></a>
+                            <a href="https://twitter.com/Solorics2"><img src="./images/skype.png" alt="twitter logo"></a>
+                            <a href="https://www.linkedin.com/in/solomon-opeyemi-0427a6155/"><img src="./images/linkdeln.png" alt="linkdeln logo"></a>
+                            <img src="./images/microverse.png" alt="microverse logo">
+                            <img src="./images/twitter2.png" alt="twitter logo">
+                        </div>
```

Comment:
- To help improve your HTML structure, I advise you wrap this in a `ul` tag instead of a `div` tag and wrap all links in an `li` tag. Please fix this 😄 😄

Reply:
_Thanks. I will work on that._